natgeo/mortar
# AEM-5956: Implement Mortar's POD component into AEM

https://jira.natgeo.com/browse/AEM-5956

These changes are in support of implementing the POD component into AEM, which was necessary because of differences in the data model

To test:

- Pull AEM-5956
- Run `npm test`
- Verify tests still pass
- Run `gulp serve`

Verify component still renders as expected at http://localhost:3000/components/podpromo/index.html and http://localhost:3000/components
